guides/hack/30-silencing-errors/05-error-codes.md

```diff
@@ -362,7 +362,7 @@ if ($y) {
 }
 ```
 
-This error is deprecated, as it was making it harder to convert
+This error is obsolete, as it was making it harder to convert
```

Comment:
IMO obsolete implies no longer relevant, or replaced. In this case, it might be better just to say 'no longer raised'